data/korath/korath missions.txt

```diff
@@ -257,3 +257,548 @@ mission "Small Scale Delivery"
+			`	This is the perfect location for life, so it's no surprise there are small plants and a few fish in the water. The seas are calmer here than the rest of the planet and an unusual formation in the ionosphere protects this spot from the worst rays of the nova. The dominant land-based lifeform seems to be toothed worms about the size of your arm.`
+			`	The strangest feature is what appears to be slow-moving islands. As one comes closer, its form is more clear: an unholy marriage between a blue whale and a giant squid. On the far end of the mesa, you spy a gravity lift system that looks suitable for lifting such a beast out of the water.`
+				decline
+
+
+
+mission "Korath [F04]: Fishy Reasoning"
+	landing
+	name "Seek answers from Iris"
+	description "Visit Iris at <planet> and ask her about Korath, fish transfers, an ocean planet, and her rainbow feather."
+	clearance "You explain the purpose of your visit, and the authorities inform you that Iris will expect you."
+	source
+		not planet "La'ab Bamen"
+	destination "La'ab Bamen"
+	to offer
+		has "First Contact: Korath Exiles: offered"
+
+		# Require a successful landing on Rabasot Pa'ah, since that gives you the whale picture.
+		has "landed on Rabasot Pa'ah"
+
+		# You have to meet Iris and know she has knowledge of Korath:
+		has "Korath [F02]: A Fishy Proposition: done"
+	on offer
+		conversation
+			`Suddenly, you have a feeling like a thousand piece jigsaw puzzle just came together before your eyes. When the Korath were drawing systems, three seemed to jump out at you. Misam, home of New Tortuga; Atik, where the La'ab Bamen black market resides; and Kor Ak'Mari, the ocean planet's system. The first fish you transferred from New Tortuga to La'ab Bamen looked like a tiny relative of the island-sized fish on the ocean planet. She had quite a lot of knowledge of the Korath, and spoke of them with admiration.`
+			`	Most curious of all is the rainbow feather on Iris's necklace, which was the exact shape of feathers on the Korath of Far'en Lei, but far more colorful.`
+			`	There's definitely something bigger going on, and it's time to track down Iris and get some answers. You last saw her on <planet>.`
+				accept
+
+
+mission "Korath [F05]: The Rainbow Feather"
+	landing
+	invisible
+	source "La'ab Bamen"
+	to offer
+		has "Korath [F04]: Fishy Reasoning: offered"
+	on offer
+		fail
+		conversation
+			`You find Iris in one of the narrow access tunnels of La'ab Bamen, observing the fish behind the glass.`
+			`	"I was wondering when you'd seek me out," she says, with a worried look. "You gave those two patrols quite a scare on Far'en Lei. They had never seen a human before, and they found one armed at their jail's lock."`
```

Comment:
We should probably vary this line based on the outcome of that encounter.  It needn't be a pass/fail thing, but accepting the prior mission should be an extra opportunity to gain favour, and Iris can clarify for you how you did on that.

Reply:
I would need you to suggest changes since I only have a 37% understanding of the meanings of the encounter.

Reply:
@roadrunner56 @ravenshining - Iris now has three responses, depending on Captain Last's actions in the first contact. Also, I added two variables:

- `First Contact: Korath Exiles: pointed gun on Far'en Lai` = took out your gun
- `First Contact: Korath Exiles: agreed to bring food` = agreed to bring food (though the player may not realize that)

Her responses:

- Took out a gun = The original message.
- Offered food = Suggests doing some jobs at La'ab Bamen.
- Neither = Shouldn't have gone there. Please don't go back.

Regardless of your actions, she doesn't want you to go back to Far'en Lai.